mlrun/utils/model_monitoring.py

```diff
@@ -1,13 +1,15 @@
+import enum
 import hashlib
 from dataclasses import dataclass
-from enum import IntEnum
 from typing import Optional
 
 import mlrun
 from mlrun.config import config
 from mlrun.platforms.iguazio import parse_v3io_path
 from mlrun.utils import parse_versioned_object_uri
 
+from ..model import ModelObj
```

Comment:
if you are outside the package scope, better to not do relative import in my opinion, meaning:
`from mlrun.model import ModelObj`